src/utils.c

```diff
@@ -374,6 +319,63 @@ SEXP coerceUtf8IfNeeded(SEXP x) {
   return(ans);
 }
 
+// class1 is used by coerseAs only, which is used by frollR.c and nafill.c only
```

Comment:
Note that there are multiple issues that are "scheduled" to be resolved by using `coerceAs`, so this list is likely to grow soon.

Reply:
why can't we use existing coerceVector?

Reply:
to handle int64 NAs

Reply:
it is also explained in https://github.com/Rdatatable/data.table/issues/3913